homeassistant/data_entry_flow.py

```diff
@@ -638,7 +641,11 @@ class FlowHandler(Generic[_FlowContextT, _FlowResultT, _HandlerT]):
 
     __progress_task: asyncio.Task[Any] | None = None
     __no_progress_task_reported = False
+    _decorated_progress_tasks: dict[str, asyncio.Task[Any]] = {}
     deprecated_show_progress = False
+    abort_reason: str = "abort"
+    abort_description_placeholders: Mapping[str, str] = MappingProxyType({})
+    progress_next_step: _FlowResultT | None = None
```

Comment:
Maybe we should make all these attributes start with double underscore to indicate they shouldn't be used by other classes?